daml-assistant/integration-tests/src/Main.hs

```diff
@@ -259,6 +262,26 @@ quickstartTests quickstartDir mvnDir = testGroup "quickstart" $
     where
         mvnRepoFlag = "-Dmaven.repo.local=" <> mvnDir
 
+quickstartScalaTests :: FilePath -> TestTree
+quickstartScalaTests quickstartDir = testGroup "quickstart-scala"
+    [ testCase "daml new" $
+          callProcessQuiet damlName ["new", projDir, "quickstart-scala"]
+    , testCase "daml build " $ withCurrentDirectory projDir $
+          callProcessQuiet damlName ["build"]
+    , testCase "sandxox and sbt" $ withCurrentDirectory projDir $
+      withDevNull $ \devNull1 -> do
```

Comment:
How does this actually work? I would have expected that you need to get the Scala dependencies from somewhere but it looks like it works automagically? Have you tried what happens if you bump the version number so that the current versions aren’t available in bintray?

Reply:
I suspect `sbt` just fetches what it needs here.

Reply:
It won’t be able to do that, if the version number is this repo is higher than the one of the last release since we haven’t uploaded the artifacts at that point.

Reply:
Right. I guess I have to do something like what you did with maven, to bundle up the dependencies and run sbt off of that bundle?

Reply:
(* or reuse the maven bundle if possible)

Reply:
Probably, I know even less about sbt than I know Maven and I know close to nothing about Maven.

Reply:
^^ then you probably know more than me at this point

I'll see what I can do.